components/frontend/src/header_footer/ViewPanel.test.js

```diff
@@ -154,10 +155,31 @@ it("shows the metrics not requiring action", () => {
 it("shows the metrics not requiring action by keypress", async () => {
     const setHideMetricsNotRequiringAction = jest.fn();
     render(<ViewPanel hideMetricsNotRequiringAction={true} setHideMetricsNotRequiringAction={setHideMetricsNotRequiringAction} />)
-    await userEvent.type(screen.getByText(/All metrics/), "{Enter}")
+    await userEvent.type(screen.getByText(/All metrics/), " ")
     expect(setHideMetricsNotRequiringAction).toHaveBeenCalledWith(false)
 })
 
+it("hides a tag", () => {
+    const toggleHiddenTag = jest.fn();
+    render(<ViewPanel tags={["security"]} toggleHiddenTag={toggleHiddenTag} />)
+    fireEvent.click(screen.getByText(/security/))
+    expect(toggleHiddenTag).toHaveBeenCalledWith("security")
+})
+
+it("hides a tag by keypress", async () => {
+    const toggleHiddenTag = jest.fn();
+    render(<ViewPanel tags={["security"]} toggleHiddenTag={toggleHiddenTag} />)
+    await userEvent.type(screen.getAllByText(/security/)[0], " ")
+    expect(toggleHiddenTag).toHaveBeenCalledWith("security")
+})
+
+it("shows a tag", () => {
+    const toggleHiddenTag = jest.fn();
+    render(<ViewPanel tags={["security"]} hiddenTags={["security"]} toggleHiddenTag={toggleHiddenTag} />)
+    fireEvent.click(screen.getAllByText(/security/)[0])
+    expect(toggleHiddenTag).toHaveBeenCalledWith("security")
```

Comment:
Do we test somewhere that the toggle actually toggles visible>hidden and hidden>visible?